### DISCUSSION TITLE
Displej 3,5

### DISCUSSION THREAD

**Discussion by Ladik12345** [mathoudebine/turing-smart-screen-python#467] [Themes]
Hi, I tried to create my own theme that would contain everything I need. But I'm not very good at graphics, if anyone has a project similar to mine but with nicer graphics, please share.

![Muj displej](https://github.com/mathoudebine/turing-smart-screen-python/assets/156787121/63756502-b220-4bca-be2a-36975504d64d)
[thema.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/14229909/thema.zip)

**Comment by Nonaud** [comment]
hello I would really like to display the FPS but on a 3.5" screen I wonder if it's possible how did you manage to display the FPS? Is this a 3.5" screen?

**Comment by Ladik12345** [comment]
Yes, it works on 3.5. Thanks to  @hicwic  and his cyberdeck project, I used his custom where it has a functional fps measurement. If you want I can share the theme.

**Reply by Nonaud** [reply]
Yes, I would love for you to share the theme. And I think it will also be useful to a lot of people. THANKS.

**Reply by Ladik12345** [reply]
Ok , přidal jsem thema i s custom senzory .

**Reply by Nonaud** [reply]
thanks i'll trie this
